DEFI/LiquidityPool/README.md

```diff
@@ -28,10 +28,12 @@ View [the multi-contract section of the user guide](https://docs.certora.com/en/
 
 
 ### Full Spec
-Run:
-```certoraRun Pool.conf```
-[A report of this run](https://prover.certora.com/output/15800/d334ef766b4e42f2a9285ed2bd12307f?anonymousKey=e46391e835ef8ffae1adc9f13d2fd3ff50a5ef2c)
-
+This example is a full spec for LiquidityPool.
+To run this use Certora cli with the conf file runFullPoll.conf
+Example of a run: https://prover.certora.com/output/1512/8fb4697781554144ac1e137fed6ca241?anonymousKey=e8a8cf904d540a3f0f8306574c3cee72dadf8d6c
+UnsatCores: https://prover.certora.com/output/1512/d76148722d6d4da98d4b050d032f3ae1/UnsatCoreVisualisation.html?anonymousKey=0df6a5352e20529c1b97a1a6d2846d9a25207b35
+Mutation test for this spec: https://mutation-testing.certora.com/?id=1dd7e71d-1451-48a9-ae17-d05a46b1581e&anonymousKey=79e13549-03d2-4436-8293-0c7d0b6b20d9
+See https://docs.certora.com for a complete guide.
```

Comment:
I think the readme needs to be changed:
1. explain the different files in the folder 
2. explain the broken version and the rule violated
3. explain the fix

Reply:
Done